services/minespace-web/src/components/Forms/incidents/IncidentForm.js

```diff
@@ -670,158 +688,171 @@ const renderUploadInitialNotificationDocuments = (
     props.formValues?.mine_determination_type_code ||
     props.formValues?.determination_type_code === "DO";
 
+  const noDocumentsInForm =
+    !props.formValues?.documents || props.formValues?.documents?.length === 0;
```

Comment:
Should documents ever be null or undefined?

Reply:
either you have not upload any document in the new incident or  the existing incident does not have any document